src/main/resources/schemas/single_justice_procedure.json

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
might be easier to read if we used 2 spaces instead of tab for indent 😄

Reply:
Updated to spaces :)